src/pages/my_registrations/MyRegistrations.tsx

```diff
@@ -44,6 +53,36 @@ export const MyRegistrations = ({ selectedUnpublished, selectedPublished }: MyRe
       return 0;
     });
 
+  const deleteDraftRegistrationMutation = useMutation({
+    mutationFn: async () => {
+      const deletePromises = draftRegistrations.map(async (registration) => {
+        const identifierToDelete = getIdentifierFromId(registration.id);
+        await deleteRegistration(identifierToDelete);
+      });
```

Comment:
kan spare oss litt jobb her også 😇 
```suggestion
      const deletePromises = draftRegistrations.map(async (registration) => await deleteRegistration(registration.identifier));
```